style.css

```diff
@@ -181,8 +198,8 @@ input[type=checkbox] {
     url("./Images/Ellipse\ 15.svg");
```

Comment:
- [ ] As required by the previous reviewer, I kindly suggest that you make sure that your images are loaded. currently your background image is not loaded because your url is pointing to an Images folder with a Capital letter `I` but your images folder is spelt with a small letter `i` instead.
<img width="951" alt="image" src="https://user-images.githubusercontent.com/52934128/201469129-8288fe36-cd07-4b74-aa23-01c3b46f5218.png">

- [ ] I kindly suggest that you can download the entire background image at once instead of downloading the icons seperately. Here is a demo video on how to do so:

https://www.loom.com/share/e6ccaab50ecc460fad7484f15f310545

Reply:
Fixed.